### ISSUE TITLE
Database creation logged multiple times

### ISSUE THREAD

**Issue by genaris** [2060-io/trust-service-resolver#8]
When I look at service initialization logs I see the following:

```
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [NestFactory] Starting Nest application...
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [InstanceLoader] PostgresModule dependencies initialized +37ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [InstanceLoader] InitDatabaseModule dependencies initialized +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM   DEBUG [DatabaseService] Database has been created succesfully
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [InstanceLoader] TypeOrmCoreModule dependencies initialized +227ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [InstanceLoader] AppModule dependencies initialized +31ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RoutesResolver] CertificationEntityController {/certification-entities} (version: 1): +41ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/certification-entities, GET} (version: 1) route +2ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/certification-entities/:id, GET} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/certification-entities, POST} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/certification-entities/:id, PUT} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/certification-entities/:id, DELETE} (version: 1) route +1ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RoutesResolver] ServiceProviderController {/service-providers} (version: 1): +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/service-providers, GET} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/service-providers/:id, GET} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/service-providers, POST} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/service-providers/:id, PUT} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/service-providers/:id, DELETE} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RoutesResolver] ReportedMessageController {/reported-messages} (version: 1): +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/reported-messages, GET} (version: 1) route +1ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/reported-messages/:id, GET} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/reported-messages/did/:did, GET} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/reported-messages, POST} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/reported-messages/:id, PUT} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/reported-messages/did/:did, PUT} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/reported-messages/:id, DELETE} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RoutesResolver] ServiceController {/services} (version: 1): +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/services, POST} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/services, GET} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/services/:id, GET} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/services/did/:did, GET} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/services/:id, PUT} (version: 1) route +1ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/services/did/:did, PUT} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [RouterExplorer] Mapped {/services/:id, DELETE} (version: 1) route +0ms
[Nest] 24210  - 03/11/2024, 8:13:29 PM   DEBUG [DatabaseService] Database has been created succesfully
[Nest] 24210  - 03/11/2024, 8:13:29 PM   DEBUG [DatabaseService] Database has been created succesfully
[Nest] 24210  - 03/11/2024, 8:13:29 PM     LOG [NestApplication] Nest application successfully started +0ms
```

Note that the line **DEBUG [DatabaseService] Database has been created succesfully** is shown 3 times. This makes me think that probably this database creation routine is being called multiple times.

Please check in NestJS module initialization that we are not calling it multiple times.

cc @gabrielmatau79

**Comment by gabrielmatau79** [comment]
I have added a pull request that resolves this issue and changes the way the InitDatabaseModule is executed

**Comment by genaris** [comment]
Fixed in #10
